wp/plugin/lib/analysis.ml

```diff
@@ -124,20 +124,10 @@ let exp_conds_mod (p : Params.t) : Env.exp_cond list =
 (* Determine which function specs to use in WP. *)
 let fun_specs (p : Params.t) (to_inline : Sub.t Seq.t)
```

Comment:
Per our Wednesday conversation: what is the default behavior if the user does not set the flag?

Reply:
The current default behavior is to set the function as called and update the stack pointer. I plan on changing this to chaosing the caller-saved registers, and if a function is one of the special functions from our verifier api, use those corresponding specs.

Reply:
👍 just make sure this appears in the comments/docs somewhere.